pages/connect.md

```diff
@@ -4,35 +4,19 @@ title: "Connecting with The Carpentries"
 permalink: "/connect/"
 ---
 
-The Carpentries is a global organisation with volunteers and learners
+HPC Carpentry is a global organisation with volunteers and learners
```

Comment:
Page title needs to change to reflect this update

Reply:
Fixed by 295ab288.